content/en/platform/corda/5.0-beta/developing/development-tutorials/build-basic-cordapp/basic-cordapp-intro.md

```diff
@@ -0,0 +1,65 @@
+---
+date: '2023-01-11'
+title: "Building Your First Basic CorDapp"
+menu:
+  corda-5-beta:
+    parent: corda-5-beta-tutorials-develop
+    identifier: corda-5-beta-tutorial-develop-building-your-first-basic-cordapp
+    weight: 1000
+section_menu: corda-5-beta
+---
+
```

Comment:
@lankydan Is this intro and story still valid for your scenarios? Anything to add here?